src/main/java/com/iri/training/model/Hobby.java

```diff
@@ -0,0 +1,20 @@
+package com.iri.training.model;
+
+public class Hobby {
+
+	private Long hobbyId;
+	private String hobbyName;
+	private String description;
+
+	public Long getHobbyId() { return hobbyId; }
+
+	public void setHobbyId(final Long hobbyId) { this.hobbyId = hobbyId; }
+
+	public String getHobbyName() { return hobbyName; }
+
+	public void setHobbyName(final String hobbyName) { this.hobbyName = hobbyName; }
+
+	public String getDescription() { return description; }
+
+	public void setDescription(final String description) { this.description = description; }
```

Comment:
Add a toString() method in all domain classes to use for logging purposes (like in the User class in the develop branch)